<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportGFXTool</class>
 <widget class="QDialog" name="ImportGFXTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Selection Preview</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="dstLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Destination Chunks</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="genColCheck">
     <property name="text">
      <string>Generate Collision</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="1" column="1">
      <widget class="QCheckBox" name="flipY">
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Flip Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="flipX">
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Flip X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="replaceButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>22</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QFrame" name="chunkView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="QListWidget" name="srcList">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>256</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="maximumSize">
      <size>
       <width>166</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>88</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Current Palette</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetPal">
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2" rowspan="4">
    <widget class="QListWidget" name="dstList">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>256</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3" rowspan="2" colspan="2">
    <widget class="QFrame" name="palFrameIn">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>128</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="srcLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Source Chunks</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QListWidget" name="importPal">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>96</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>96</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="gridSize">
      <size>
       <width>17</width>
       <height>16</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Missing Import Colors</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportGFXTool</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportGFXTool</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
